### PR TITLE
feat(KUI-1262): upgrade to KTH Style 10

### PR DESCRIPTION
**Pull request by karlandindrakryggen**
Main goal of this PR is to upgrade KTH Style to version 10. 

A short summary of what have been done:
* Upgrading KTH packages and adding new `@kth/style` along side with old `kth-style` (the old will be deprecated)
* General cleanup of DOM structure to algin with other Om kursen apps. 
  * More cleanup in side-panel. **Take an extra look at** `RoundInformationOneCol.jsx`
* New header and footer
* New of style of all Buttons, Alerts and SideMenu
* Added "prose" style to articles/text sections
* General cleanup of style including font-family, colors and more.
* Move of toolbar url (Personliga menyn) to parameters file to support ref version in ref/loaclhost

Some code are duplicated a cross different Om kursen repos. Such SCSS has been moved to `shared.scss` and components to `/shared-components`. PRs for other Om kursen apps that are upgraded at the same time with same structure:
* Kopps public: https://github.com/KTH/kopps-public-react-web/pull/242
* Kurs PM: https://github.com/KTH/kurs-pm-web/pull/373
* Kursutveckling: https://github.com/KTH/kursutveckling-web/pull/274

### CODE REVIEW

File: public/js/app/components/__tests__/SideMenu.test.js [hunk 1]
```diff
@@ -21,21 +21,25 @@ describe('Component <SideMenu>', () => {
...
     expect(links[3].href).toBe('http://localhost/kursutveckling/KIP1111?l=en')
-    expect(links[3].title).toBe('Course development')
+    expect(links[3]).toHaveTextContent('Course development')
 
     expect(links[4].href).toBe('http://localhost/kursutveckling/KIP1111/arkiv?l=en')
-    expect(links[4].title).toBe('Archive')
+    expect(links[4]).toHaveTextContent('Archive')
 
-    expect(screen.getByText('About course KIP1111')).toBeInTheDocument()
+    // const a screen.getAllByText('About course KIP1111')
```
Comment (karlandindrakryggen) [inline public/js/app/components/__tests__/SideMenu.test.js:38]: ta bort

File: server/views/layouts/publicLayout.handlebars [hunk 2]
```diff
@@ -24,14 +25,13 @@
...
   {{prefixScript '/static/kth-style/js/backtotop.js' 'head-scripts'}}
 
-  {{#extend 'head-scripts'}}
-    <script src="//www.kth.se/social/toolbar/widget.js"></script>
-  {{/extend}}
-
   {{{render 'head-scripts'}}}
+  
+  <script src="{{toolbarUrl}}"></script>
+  <script src="{{toolbarUrl}}"></script>
```
Comment (karlandindrakryggen) [inline server/views/layouts/publicLayout.handlebars:33]: ups

File: public/js/app/components/RoundInformationOneCol.jsx [hunk 3]
```diff
@@ -44,185 +43,155 @@ function RoundInformationOneCol({
...
+
+              <h3>{translation.courseRoundInformation.round_tutoring_language}</h3>
+              <p>{round ? round.round_tutoring_language : missingInfoLabel}</p>
+              <h3>{translation.courseRoundInformation.round_course_place}</h3>
+              <p>{round ? round.round_course_place : missingInfoLabel}</p>
+              <h3>
+                {translation.courseRoundInformation.round_max_seats}
+                {round && round.round_seats && (
+                  <InfoModal
+                    parentTag="h3"
```
Comment (belanglos) [inline public/js/app/components/RoundInformationOneCol.jsx:116]: Was `parentTag` not removed as prop?

**Review by belanglos — APPROVED** [review]
LGTM, only found one unused prop.

File: public/js/app/components/RoundInformationOneCol.jsx (hunk 3 above)
Comment (karlandindrakryggen) [reply public/js/app/components/RoundInformationOneCol.jsx]: Good catch! Fixed now.

**Review by belanglos — APPROVED** [review]

**Pull request merged**